image/config.go

```diff
@@ -70,7 +70,7 @@ func (c *config) runtimeSpec(rootfs string) (*specs.Spec, error) {
 	}
 
 	var s specs.Spec
-	s.Version = "0.5.0"
+	s.Version = specs.Version
```

Comment:
this was the runtime version not the image one right? if so this is wrong (do image and runtime follow same release? if so this is still wrong to me, but righter...)